src/oc/hoc.cpp

```diff
@@ -713,12 +715,12 @@ void hoc_execerror_mes(const char* s, const char* t, int prnt){	/* recover from
 	ctp = cbuf;
 	*ctp = '\0';
 
-	if (oc_jump_target_ && nrnmpi_numprocs_world == 1) {
+	if (oc_jump_target_ && (nrnmpi_numprocs_world == 1 || !nrn_mpiabort_on_error_)) {
 		hoc_newobj1_err();
 		(*oc_jump_target_)();
 	}
 #if NRNMPI
-	if (nrnmpi_numprocs_world > 1) {
+	if (nrnmpi_numprocs_world > 1 && nrn_mpiabort_on_error_) {
```

Comment:
```-mpi``` has the effect of turning off x11 graphics (equivalent to -nogui), but otherwise there is not much difference between one mpi rank and not using mpi. I.e in both cases pc.nhost_world() == pc.nhost_bbs() == pc.host() == 1 . There is an internal global variable called ```nrnmpi_use``` which is 1 if nrnmpi_init has been called.

Reply:
If ```nrnmpi_numprocs_world == 1``` then we do NOT call MPI_Abort on error under either interpretation of MPI or non-MPI execution.

Reply:
Ok, that clarifies my original question.